src/lib.rs

```diff
@@ -2,6 +2,7 @@
 #![feature(c_unwind)]
 #![feature(naked_functions)]
 #![feature(non_exhaustive_omitted_patterns_lint)]
+#![feature(cfg_target_abi)]
```

Comment:
Is there a way to achieve this without additional nightly features? I'd like to avoid adding extra ones.

Reply:
I think it might be doable with `target_feature`, but I'm not sure. Will do some testing and find out.

Reply:
Never mind, the counterparts in `target_feature` are still unstable. The other option would be a cargo feature, but that feels weird. Your choice I guess.

Reply:
cfg_target_abi stabilized in the recent nightly (https://github.com/rust-lang/rust/pull/119590), so we can now remove this `#![feature]`